### PR TITLE
Swap UI update

### PR DESCRIPTION
**Pull request by Rockindash**
Swap Transaction updated as per new UI Design

### CODE REVIEW

File: VultisigApp/VultisigApp/Views/Swap/SwapTransaction.swift [hunk 1]
```diff
@@ -58,6 +58,30 @@ class SwapTransaction: ObservableObject {
...
+    var gasInReadable: String {
+        guard var decimals = Int(fromCoin.decimals) else {
+            return .empty
+        }
+        if fromCoin.chain.chainType == .EVM {
+            // convert to Gwei , show as Gwei for EVM chain only
+            guard let weiPerGWeiDecimal = Decimal(string: EVMHelper.weiPerGWei.description) else {
+                return .empty
+            }
+            return "\(Decimal(gas) / weiPerGWeiDecimal) \(fromCoin.feeUnit)"
```
Comment (flypaper0) [inline VultisigApp/VultisigApp/Views/Swap/SwapTransaction.swift:71]: Gas for SwapTransaction represents total fee needed (GasLimit * gasPrice). 

To get gas you looking for we should fetch ChainSpecific from BlockChainService and get gas property 

```
let chainSpecific = try await blockchainService.fetchSpecific(
    for: tx.fromCoin,
    action: .swap,
    sendMaxAmount: false
)
```

File: VultisigApp/VultisigApp/Views/Swap/SwapTransaction.swift [hunk 2]
```diff
@@ -58,6 +58,30 @@ class SwapTransaction: ObservableObject {
...
+        }
+        
+        // If not a native token we need to get the decimals from the native token
+        if !fromCoin.isNativeToken {
```
Comment (flypaper0) [inline VultisigApp/VultisigApp/Views/Swap/SwapTransaction.swift:75]: feeCoin(tx:) from SwapCryptoViewModel can be used to get nativeCoin

File: VultisigApp/VultisigApp/Views/Swap/SwapTransaction.swift (hunk 1 above)
Comment (johnnyluo) [reply VultisigApp/VultisigApp/Views/Swap/SwapTransaction.swift:71]: image
Check the gas , that's not correct

**Review by enriquesouza — DISMISSED** [review]

**Review by johnnyluo — APPROVED** [review]
LGTM

**Pull request merged**